File: Lab15/lab15.py
from math import *
def f(i):
    return sqrt(i+2) - sqrt(i) - 3/11

def g(j):
    return ((3*(j**2) - 11) / ((-j**3) - 2))
n = 10  
a = [[0]*n for _ in range(n)]
total_sum = 0
for i in range(n):
    for j in range(n):
        a[i][j] = f(i+1) + g(j+1) 
        total_sum += a[i][j]

tot = 1/100 * total_sum
print(round(tot, 6))

# print(f(10))
# print(g(10))
# import time
# start_time = time.time()
# end_time = time.time()
# execution_time = end_time - start_time
# print(f"Execution time: {execution_time} seconds")

File: Lab15/lab15.c
#include <stdio.h>
#include <math.h>

double f(double i) {
    double drob;
    drob = (double)3/11;
    return sqrt(i+2) - sqrt(i) - drob;
}

double g(double j) {
    return ((3*pow(j,2)) - 11) / (pow((-j), 3) - 2);
}

int main() {
    int n = 10;  
    double a[n][n];  
    double total_sum = 0;  

    
    for (int i = 0; i < 10; i++) {
        for (int j = 0; j < 10; j++) {
            a[i][j] = f(i+1) + g(i+1);
            total_sum += a[i][j];  
        }
    }
    double total;
    double ch;
    ch = (double)1/100;
    total = (ch*total_sum);

    printf("%lf\n", total);
   
    return 0;
}
